### DISCUSSION TITLE
localized cookie consent

### DISCUSSION THREAD

**Discussion by mcg-pimcore** [pimcore/pimcore#10279] [Q&A / Support]
Hello,

How can I add localized cookie consent and after user consent how can I save visitor info(ipaddress and other detail).
Is there any plugin or setting for cookie consent or I have to do it manually.

**Comment by NiklasBr** [comment]
We use https://github.com/ConnectHolland/cookie-consent-bundle/  -- it has support for localization and logging of consent.

**Reply by mcg-pimcore** [reply]
Hello Nikalas, Thanks for reply. One more question, can I change consent form according to wish.

**Reply by NiklasBr** [reply]
You can see in the README that you can change a lot of things like:

- Categories
- Javascript & Events
- Template Themes & Template override
- CSS Styling
- Translations

**Reply by mcg-pimcore** [reply]
OK

**Reply by dpfahlbusch** [reply]
@NiklasBr short question: how did you get the logging to work?

**Reply by NiklasBr** [reply]
@dpfahlbusch I also found it a bit confusing, but found this issue which explains it https://github.com/ConnectHolland/cookie-consent-bundle/issues/63

**Reply by NiklasBr** [reply]
I forgot/remembered you also need to run `doctrine:schema:update --force` to create the db table.
